stations_validator.py

```diff
@@ -0,0 +1,8 @@
+import pandas as pd 
+
+class StationsValidator():
```

Comment:
Hmm... this is the first time I've seen the tests wrapped inside a class definition. Perhaps I'm missing some knowledge from the world of standard software engineering practices. Is there a reason this is done? Would the original form be simpler than this, or is there extra function offered by wrapping the tests in a class?